src/config/defaults.ts

```diff
@@ -57,6 +65,7 @@ export interface FaviconOptions {
   readonly relatedApplications?: Application[];
   readonly icons?: Dictionary<IconOptions | boolean | string[]>;
   readonly files?: Dictionary<FileOptions>;
+  readonly shortcuts?: Dictionary<ShortcutOptions>;
```

Comment:
Type is `Dictionary<>` but all examples use arrays. It seems like correct type should be `ShortcutOptions[]`.

Reply:
Oops forget to change that.